prow/config/org/org.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package org
+
+import (
+	"fmt"
+)
+
+// OrgMetadata declares metadata about the github org.
+//
+// See https://developer.github.com/v3/orgs/#edit-an-organization
+type OrgMetadata struct {
+	BillingEmail                 *string              `json:"billing_email,omitempty"`
+	Company                      *string              `json:"company,omitempty"`
+	Email                        *string              `json:"email,omitempty"`
+	Name                         *string              `json:"name,omitempty"`
+	Description                  *string              `json:"description,omitempty"`
+	Location                     *string              `json:"location,omitempty"`
+	HasOrganizationProjects      *bool                `json:"has_organization_projects,omitempty"`
+	HasRepositoryProjects        *bool                `json:"has_repository_projects,omitempty"`
+	DefaultRepositoryPermission  *RepoPermissionLevel `json:"default_repository_permission,omitempty"`
+	MembersCanCreateRepositories *bool                `json:"members_can_create_repositories,omitempty"`
+}
+
+// Org declares metadata as well as its people and teams.
+type Org struct {
```

Comment:
Doesn't `go lint` complain when you don't use the package name as a namespace? Thiss could be `org.Config` and below could be `org.Metadata`

Reply:
Done